include/vrapp/VRVolumeApp.h

```diff
@@ -319,5 +319,8 @@ class VRVolumeApp
 
   APPMODE m_app_mode;
 
+  float m_global_min;
+  float m_global_max;
```

Comment:
Are these globals for the class or the whole application? Do they ever change while the program is running?

Reply:
I renamed them to reflect their true meaning.